utils/ICutils.py

```diff
@@ -3,6 +3,7 @@
 and saved into a directory for further analysis. 
 """
 
+import logging
 import sys
 import numpy as np
 import pathlib
```

Comment:
```suggestion
import pathlib
import logging
import sys
import numpy as np
```
reformatting to follow Python's [PEP8](https://peps.python.org/pep-0008/#imports) standards

Reply:
👍 Changed, I will need to get used to this!